Description: Context Free ships with a local copy of libagg which it uses by
 default. The Makefile can be configured to use a system copy of
 libagg and this patch makes this change to the Makefile.
Author: John Horigan <john@glyphic.com>
Forwarded: not-needed

--- a/Makefile
+++ b/Makefile
@@ -98,14 +98,14 @@
 #
 # Configuration for local AGG
 #
-SRCS += $(AGG_SRCS)
-INC_DIRS += $(AGG_DIR) $(AGG_DIR)/agg2
+#SRCS += $(AGG_SRCS)
+#INC_DIRS += $(AGG_DIR) $(AGG_DIR)/agg2
 
 
 #
 # Configuration for system AGG
 #
-#LIBS += agg
+LIBS += agg
 
 
 OBJS = $(patsubst %.cpp,$(OBJ_DIR)/%.o,$(SRCS))
